metrics/report/report_dockerfile/Dockerfile

```diff
@@ -9,29 +9,29 @@
 #  - pandoc
 #  - The report generation R files and helper scripts
 
-# I would have liked to have used from base rocker/verse, but that
-# does not have all the tools we need, and it seems that apt install
-# does not simply work - texlive for instance did not install.
-#
-# So, build up our image by hand then...
-FROM ubuntu
+# Start with the base rocker tidyverse.
+# We would have used the 'verse' base, that already has some of the docs processing
+# installed, but I could not figure out how to add in the extra bits we needed to
+# the lite tex version is uses.
+FROM rocker/tidyverse
 
 # Version of the Dockerfile
-LABEL DOCKERFILE_VERSION="1.0"
+LABEL DOCKERFILE_VERSION="1.1"
 
 # Without this some of the package installs stop to try and ask questions...
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && \
-	apt-get install -y tzdata r-base pandoc texlive
-
-# R packages get built from source, and can take some time, so split them
-# into individual RUNs so we can make some later modifications without maybe taking
-# the full image rebuild hit.
-RUN Rscript -e "install.packages('tidyverse')"
-RUN Rscript -e "install.packages('knitr')"
-RUN Rscript -e "install.packages('gridExtra')"
-RUN Rscript -e "install.packages('ggpubr')"
+# Install the extra doc processing parts we need for our Rmarkdown PDF flow.
+RUN apt-get update -qq && \
+  apt-get install -y \
```

Comment:
change spaces to tabs here and in the lines below...

Reply:
:-)
 I was taking the style/inspiration from the `tidyverse` Dockerfile this is layed over - at https://github.com/rocker-org/rocker-versioned/blob/master/tidyverse/Dockerfile#L3-L21
Is there, or do we, have a standard for our Dockerfiles - tabs, 2-space-yaml-like etc.?
I'm happy to change if there is some standard we are adhering to?

Reply:
Specifically for Dockerfiles we haven't set a standard, but I think we have chosen to use tabs instead of spaces for all scripts. 
Although taking a look at other Dockerfiles in this repo, I see that they use 4-space indents... so now I am not sure.

Reply:
arf! - thanks for checking @chavafg - let me see if I can go find any global guidance out on t'intinet....

Reply:
took a look at this page: https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/ but it does not says anything about indenting Dockerfiles and in their examples they use spaces, so I think this is ok ;-)